Jenkinsfile

```diff
@@ -5,6 +5,15 @@ pipeline {
     parameters {
         booleanParam(name: 'UI', defaultValue: true)
         string(name: 'Box', defaultValue: 'default', description: 'Vagrant Box')
+        string(name: 'HOOT_UI_AWS_ENV_TAG', defaultValue: 'testing')
```

Comment:
@gvoltz Same comment as with hootenanny.  Do we want to make these configurable parameters?

Reply:
It was more of a clutter vs. hard-coding in my mind for the tags in the Jenkinsfile. The params iI'd bet would rarely be changed, but without the params the tags would be hard-coded from the environment namespace which I wanted to avoid.